askup/admin.py

```diff
@@ -90,20 +91,40 @@ def queryset(self, request, queryset):
             return queryset.filter(qset_id=self.value())
 
 
+class DomainsInline(TabularInline):
+    model = Domain
+    can_delete = True
+    verbose_name_plural = 'Domains'
+    fk_name = 'organization'
+    extra = 0
+
+
 class OrganizationAdmin(admin.ModelAdmin):
     """Admin view for the Organization model."""
 
     form = OrganizationModelForm
-    list_display = ('name', 'email_patterns')
+    list_display = ('name', 'domains')
     fields = (
         'name',
         'users',
     )
+    inlines = (DomainsInline,)
+
+    def get_inline_instances(self, request, obj=None):
+        """
+        Return an inline instances for the form.
+
+        Overrides the function of the ModelForm with the same name.
+        """
+        if not obj:
+            return list()
+
+        return super().get_inline_instances(request, obj)
 
     @staticmethod
-    def email_patterns(obj):
+    def domains(obj):
         """Return email-patters of the organization comma separated."""
-        return ", ".join(tuple(pattern.text for pattern in obj.emailpattern_set.all()))
+        return ", ".join(tuple(domain.name for domain in obj.domain_set.all()))
```

Comment:
the tuple is not needed we can just use comprehension generator here

Reply:
done